pkg/start/start.go

```diff
@@ -37,33 +38,19 @@ func ensureNerdctlArchiveCache(y *limayaml.LimaYAML) (string, error) {
 	}
 
 	errs := make([]error, len(y.Containerd.Archives))
-	for i := range y.Containerd.Archives {
-		f := &y.Containerd.Archives[i]
-		if f.Arch != *y.Arch {
-			errs[i] = fmt.Errorf("unsupported arch: %q", f.Arch)
-			continue
-		}
-		logrus.WithField("digest", f.Digest).Infof("Attempting to download the nerdctl archive from %q", f.Location)
-		res, err := downloader.Download("", f.Location, downloader.WithCache(), downloader.WithExpectedDigest(f.Digest))
+	for i, f := range y.Containerd.Archives {
+		path, err := fileutils.DownloadFile("", f, "the nerdctl archive", *y.Arch)
 		if err != nil {
-			errs[i] = fmt.Errorf("failed to download %q: %w", f.Location, err)
+			errs[i] = err
 			continue
 		}
-		switch res.Status {
-		case downloader.StatusDownloaded:
-			logrus.Infof("Downloaded the nerdctl archive from %q", f.Location)
-		case downloader.StatusUsedCache:
-			logrus.Infof("Using cache %q", res.CachePath)
-		default:
-			logrus.Warnf("Unexpected result from downloader.Download(): %+v", res)
-		}
```

Comment:
?

Reply:
It was moved to pkg/downloader (previously it was duplicated here)